testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -1900,6 +1938,26 @@ public void updateRowRaw() throws IOException {
     @SuppressWarnings("unchecked")
     Map<String, String> data = objectMapper.readValue(body, Map.class);
     assertThat(data).containsAllEntriesOf(rowUpdate);
+
+    // Also verify that we can "delete" lastName
```

Comment:
(that is, set field to `null` with update -- exercises null usage via different path)